src/allotropy/allotrope/models/shared/definitions/definitions.py

```diff
@@ -52,9 +52,15 @@ class TStatisticDatumRole(Enum):
     minimum_value_role = "minimum value role"
 
 
+class TNonNumericQuantityValue(Enum):
+    nan = "NaN"
+    plus_infinite = "+Infinity"
+    minus_infinite = "-Infinity"
+
```

Comment:
```suggestion
    NaN = "NaN"
    Infinity = "Infinity"
    field_Infinity = "-Infinity"
```

Reply:
Requesting this change because this is how the schema generator names these values, want to keep as close to auto-generated as possible.

Reply:
@nathan-stender @nathan-stender Is it ok to change `+Infinity` to just `Infinity?

keeping the `+Infinity` I got:
```python
class ValueEnum(Enum):
    NaN = "NaN"
    field_Infinity = "+Infinity"
    field_Infinity_1 = "-Infinity"
```